resources/assets/js/videos/components/videoScreen/videoPlayback.vue

```diff
@@ -24,6 +24,8 @@ export default {
             // Allow a maximum of 100x magnification. More cannot be represented in the
             // URL parameters.
             minResolution: 0.01,
+            // parameter tracking seeking state specific for frame jump, needed because looking for seeking directly leads to error
+            seekingFrame: this.seeking,
```

Comment:
Can you please explain why you can't use `this.seeking` directly or what error would be caused? Also I don't think you can use `this.seeking` here already.

Reply:
Odly it seems that we don't get the same behaviour: when I hold the keys pressed I can see the frames rendering as seeking goes along... Also if I use `this.seeking` below instead of `seekingFrame` and hold the keys, the spinning wheel stays stuck at some point, as if we never leave the seeking state.

I'm running my tests on a local address on my pc that I open in Chrome v125.0.6422.144 (Windows 64 bits)
For the projet installation I followed these instructions: [https://github.com/biigle/core/blob/master/DEVELOPING.md](url)

Reply:
Yes I still get the same behavior...

I tried on chromium on WSL and I am able to see the same behavior as you, so it is indeed a linux vs windows difference.

At this point I have no better idea than warning windows users it might get stuck if using keyboard and that they may have to refresh the page. What do you think ?

Reply:
Ok if this is a Windows issue and it works with your `seekingFrame` then we can keep it that way!

I also now checked if you can use `this.seeking` in `data()` like that and you can! So this conversation is resolved.